File: articles/human-in-the-loop-and-human-judgment/fortran/review_capacity_score.f90
program review_capacity_score
  implicit none
  real, dimension(5) :: scores
  real :: capacity
  scores = (/0.56, 0.62, 0.58, 0.60, 0.48/)
  capacity = sum(scores) / 5.0
  print *, "review_capacity_score=", capacity
end program review_capacity_score

File: articles/human-in-the-loop-and-human-judgment/rust/review_capacity_score.rs
fn main() {
    let scores = [0.56_f64, 0.62_f64, 0.58_f64, 0.60_f64, 0.48_f64];
    let total: f64 = scores.iter().sum();
    let capacity = total / scores.len() as f64;
    println!("review_capacity_score={:.4}", capacity);
}
